data/templates/logs/logrotate/vyos-rsyslog.tmpl

```diff
@@ -0,0 +1,13 @@
+/var/log/messages {
+    create
+    missingok
+    nomail
+    notifempty
+    rotate {{ rotate|default('10') }}
+    size {{ max_size|default('1') }}M
```

Comment:
As there is a `<defaultValue>` specified on the XML implementation the `default()` statement is superfluous.